packages/typespec-azure-resource-manager/src/rules/arm-resource-provisioning-state-rule.ts

```diff
@@ -9,7 +10,7 @@ export const armResourceProvisioningStateRule = createRule({
   description: "Check for properly configured provisioningState property.",
   messages: {
     default:
-      "The RP-specific property model in the 'properties' property of this resource must contain a 'provisioningState property.  The property type should be an enum, and it must specify known state values 'Succeeded', 'Failed', and 'Canceled'.",
+      "The RP-specific property model in the 'properties' property of this resource must contain a 'provisioningState property.  The property type should be an enum or a union of string values, and it must specify known state values 'Succeeded', 'Failed', and 'Canceled'.",
```

Comment:
Also please have an issue for updating all examples to union if not already. Thx